src/chain-libs/chain-impl-mockchain/src/ledger/tests/transaction_tests.rs

```diff
@@ -41,74 +40,7 @@ pub fn transaction_fail_when_255_outputs() {
             expected: 254,
             actual: 255
         }),
-        test_ledger.apply_transaction(fragment, BlockDate::first())
-    );
-}
-
-#[test]
-pub fn transaction_fail_when_validity_out_of_range() {
```

Comment:
should we have some type of mempool test to replace these? are TXs flushed or do they backlog?

Reply:
I have not removed mempool tests related to the `expired_date` field they are still exist. As ledger validation has been removed these tests are unneeded anymore.